docker/Dockerfile.arm64

```
@@ -1,4 +1,4 @@
-FROM alpine:3.16 AS builder
+FROM alpine:3.18 AS builder
 RUN apk add --no-cache 'crystal=1.4.1-r0' shards sqlite-static yaml-static yaml-dev libxml2-dev zlib-static openssl-libs-static openssl-dev musl-dev
```

The crystal version needs to be updated:
https://pkgs.alpinelinux.org/packages?name=crystal&branch=v3.18&arch=aarch64

```suggestion
RUN apk add --no-cache 'crystal=1.8.2-r0' shards sqlite-static yaml-static yaml-dev libxml2-dev zlib-static openssl-libs-static openssl-dev musl-dev
```

---

Do we validate that it works great on crystal 1.8?

---

I've been running the test instance with it for quite a while, yes.
Also we already have the 1.8 branch in the CI matrix.

---

ok cool :)